### ISSUE TITLE
New user email > misleading/incorrect email

### ISSUE THREAD

**Issue by alisonjo2786** [Drupal4Gov/Drupal-GovCon-2017#279]
### Requirements
- [ ] Change text in email sent to "new user" immediately following user creating new website account: Completely replace the current email body with something like...

> ajm524,
> 
> Thank you for creating a user account on drupalgovcon.org!  Now you can register for your Drupal GovCon 2017:
> [link to register]
> 
> Not sure if you already have an event ticket?  Check your user profile:
> [link to "my tickets"]
> 
> Going forward, you can log into the Drupal GovCon website at [link] using:
> username: ajm524
> password: Your password
> 
> _[**And,** do we have some sort of standard sign-off, with like, "Questions? Contact..." and "Sincerely, The Drupal GovCon team"? -- @jdearie Does that all sound okay? ]_

#### Background
TL;DR: New user email is misleading b/c you've already set a password and you're already logged-in.

When I created a new user account, I got an email that said "Thank you for registering at Drupal GovCon. You may now log in by clicking this link or copying and pasting it into your browser" etc. -- but when I created my account, I had already set a password, and I was already logged-in (as the success message affirmed, "Registration successful, you are now logged in").

![image](https://cloud.githubusercontent.com/assets/2660348/25026935/500d5eb4-2077-11e7-9608-4e2034914b76.png)

**Comment by aweingarten** [comment]
@alisonjo2786 what would be less misleading?

**Comment by alisonjo2786** [comment]
@aweingarten The reason it's misleading is that you've already set a password and you're already logged-in.

So, like,

You've created an account on drupalgovcon.org.  Now you can go ahead and register for the event! [link to register]

Going forward, you can log in at [link] using:
username: ajm524
password: Your password

(or similar) ( @jdearie )

**Comment by alisonjo2786** [comment]
@aweingarten High priority
